src/Doctrine/EntityClassGenerator.php

```diff
@@ -50,6 +50,7 @@ public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $
                 'api_resource' => $apiResource,
                 'should_escape_table_name' => $this->doctrineHelper->isKeyword($tableName),
                 'table_name' => $tableName,
+                'is_typed' => $isTyped,
```

Comment:
I don't think we need this - there should already be a `use_typed_properties` variable that's added automatically https://github.com/symfony/maker-bundle/blob/a47408fa6e39034a5abe324a46f4e6733266267a/src/Generator.php#L171

We can use that already in the template